src/tools/compiletest/src/main.rs

```diff
@@ -841,6 +851,48 @@ fn is_android_gdb_target(target: &String) -> bool {
     }
 }
 
+/// Returns `true` if the given target is a MSVC target for the purpouses of CDB testing.
+fn is_pc_windows_msvc_target(target: &String) -> bool {
+    target.ends_with("-pc-windows-msvc")
+}
+
+fn find_cdb(target: &String) -> Option<OsString> {
+    if cfg!(windows) && is_pc_windows_msvc_target(target) {
```

Comment:
Could this be inverted to reduce indentation? e.g.:

```rust

if !current_condition {
    return None;
}

// ...
```

Reply:
Done!